tensorflow/lite/micro/kernels/BUILD

```diff
@@ -90,6 +90,43 @@ cc_library(
     ],
 )
 
+cc_library(
+    name = "depthwise_conv",
```

Comment:
this should depend on :conv for the conv.h, right?

Reply:
It does below in the deps section, fixed by my second commit